Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -956,6 +956,8 @@ extension _FileManagerImpl {
                 #if os(WASI)
                 // WASI does not support extended attributes
                 throw CocoaError.errorWithFilePath(.featureUnsupported, path)
+                #elseif canImport(Android)
+                // Android doesn't allow setting this for normal apps, so just skip it.
```

Comment:
If Android doesn't allow setting this, should we throw an error similarly to WASI, either `.featureUnsupported` or perhaps an error indicating that the operation isn't allowed?

Reply:
I considered simply enabling the WASI error block above on Android too, and would have if this method were just setting these extended attributes alone, but this `setAttributes()` method is also setting a bunch of other attributes and works fine for that on Android.

I don't know the situation on WASI but pretty much all files on Android appear to have some preset extended attributes, but normal users can't change them. Given that, it makes sense to just skip setting them on Android, whereas maybe WASI normally doesn't have them or something, so the error above is rare on that platform.